O365DataRetriever.ps1

```diff
@@ -647,6 +647,16 @@ $ConnectButton.Add_Click( {
 		}
 		else{
 			Write-Host "Microsoft Online Module present. OK." -ForegroundColor Green
+        }
+        
+        #Check if Azure AD Powershell module present
+		if (!(Get-Module -ListAvailable -Name "AzureAD")){
```

Comment:
Appreciate this will come at a later stage. As I mentioned in the pre-reqs _(README file)_, I'm not using the AzureAD cmdlets yet. This will require a re-write of all the Msol cmdlets after checking they can do the same _(which I doubt at this point in time)_, or even make the transition gradually. So you leave this one out for now...

Reply:
Commented out for later use. (hope this is ok)

Reply:
Just remove it completely please. We'll add it when we start changing the cmdlets.